### PR TITLE
Disabled WeakEventManager for WPF

### PR DESCRIPTION
**Pull request by rhautefeuille**
Fixed lost INPC subscriptions
Basically this fix assumed Net45 == WPF but it isn't true for this fork
https://github.com/reactiveui/ReactiveUI/issues/661

### CODE REVIEW

File: ReactiveUI/ReactiveCommand.cs [hunk 1]
```diff
@@ -281,7 +281,7 @@ public static ReactiveCommand<object> CreateCombined(params IReactiveCommand[] c
     /// </summary>
     public class ReactiveCommand<T> : IReactiveCommand<T>, IReactiveCommand
     {
-#if NET_45
+#if NET_40
```
Comment (gluck) [inline ReactiveUI/ReactiveCommand.cs]: shouldn't it be NET_40 *or* NET_45 (so we keep the NET_45 build correct) ?

Comment (rhautefeuille) [reply ReactiveUI/ReactiveCommand.cs]: I thought this fork only purpose was to support net40.
If we need to support net45 as well, I can specify both, sure.

Comment (gluck) [reply ReactiveUI/ReactiveCommand.cs]: I don't think we broke .Net4.5 when we did it :)

**Comment by rhautefeuille** [comment]
done

**Pull request merged**